tox.ini

```diff
@@ -1,6 +1,6 @@
 [tox]
 envlist =
-    py{38,39,310,311}-test{,-alldeps,-devdeps,-predeps}{,-cov}
+    py{38,39,310,311}-test{,-alldeps,-devdeps,-predeps}{-romandeps}{,-cov}
```

Comment:
I've added a `romandeps` testing option, and a CI test that uses it (allowed_fail=True), but I'm not 100% sure I've done this sensibly. Tips welcome!